src/AutoRest.CSharp/Common/Output/Models/Types/ObjectTypeConstructor.cs

```diff
@@ -15,7 +15,7 @@ public ObjectTypeConstructor(string name, MethodSignatureModifiers modifiers, Pa
         {
             Signature = new ConstructorSignature(
                 name,
-                $"Initializes a new instance of {name}",
+                $"Initializes a new instance of <see cref=\"{name}\"/>",
```

Comment:
LGTM, but do you think that we need to use cref in the ctor of rest operation and lro as well, even if they are internal?